Notebooks/FitMagProfile.ipynb

```diff
@@ -155,9 +184,29 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x16a190c0dd8>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "Q = Mag.fitline(Box)\n",
+    "Q = Simulator.fitline(model,survey)\n",
```

Comment:
![fitmagprofile 2017-06-28 12-24-51](https://user-images.githubusercontent.com/13682747/27656168-d8a7b33a-5bfc-11e7-9fc2-65696089db85.png)

The legend is far from the actual plot